ecl/hql/hqlpmap.cpp

```diff
@@ -990,7 +990,13 @@ static bool isTrivialTransform(IHqlExpression * expr, IHqlExpression * selector,
                         return false;
                     IHqlExpression * rightField = rhs->queryChild(1);
                     if (leftField != rightField)
-                        return false;
+                    {
+                        if (!allowDeserialize)
+                            return false;
+                        //MORE
+                        if (true)
```

Comment:
@ghalliday Don't understand what this code is trying to do here.  Will new code be replacing this very soon?

Reply:
In a subsequent PR it will allow serialized->deserialized if the flag is passed correctly.